app/scripts/filter/text.js

```diff
@@ -77,10 +77,43 @@ const filters_text = angular.module('filters_text', [])
     return function(timestamp) {
       var is_valid = timestamp == 0 || (new Date(timestamp)).getTime() > 0;
       if (!is_valid) {
-        return "Invalid Timestamp";
+        return 'Invalid Timestamp';
       }
       return new Date(timestamp).toISOString();
     };
+  })
+  .filter('formatTimeWithTimezone', function() {
+    function formatTimeElement(element) {
+      if (element < 10) {
+        return '0' + Math.floor(element);
+      } else {
+        return Math.floor(element);
```

Comment:
Would it make sense to force a string conversion in the second branch too, to avoid having different types returned?

Reply:
Yeah, it'd be good for consistencies sake :+1: